### PR TITLE
update travis configuration to use the standard ROS repositories path

### PR DESCRIPTION
**Pull request by ipa-nhg**

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
@ipa-nhg fyi: I've updated the base to `kinetic-devel` instead of the branch I used for #210.

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
+sudo: required
+dist: trusty
+language: generic
+
 env:
   matrix:
-    - ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu NOT_TEST_BUILD=true NOT_TEST_INSTALL=true
-    - ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu NOT_TEST_BUILD=true NOT_TEST_INSTALL=true
-    - ROS_DISTRO="kinetic"  ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu NOT_TEST_INSTALL=true
-    - ROS_DISTRO="kinetic"  ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu NOT_TEST_INSTALL=true
```
Comment (gavanderhoorn) [inline .travis.yml]: I can't be sure yet, but I believe there was a reason for the `NOT_TEST_INSTALL` and other env vars there.

Comment (ipa-nhg) [reply .travis.yml]: I tested this configuration locally (docker container) and worked.. I can re-add the `NOT_TEST_INSTALL=true` but I think is not needed anymore

Comment (gavanderhoorn) [reply .travis.yml]: Ok. Let's remove it then and keep an eye on it.

We can always re-add it later.

**Comment by gavanderhoorn** [comment]
Somehow I cannot get Travis to build this PR.

This PR is also not listed on the Travis admin interface.

Could you perhaps resubmit the PR @ipa-nhg?

**Comment by gavanderhoorn** [comment]
Replaced by #229.

**Pull request closed without merging**